core/src/main/java/tachyon/command/TFsShell.java

```diff
@@ -470,6 +514,8 @@ public void printUsage() {
     System.out.println("       [pin <path>]");
     System.out.println("       [unpin <path>]");
     System.out.println("       [free <file path|folder path>]");
+    System.out.println("       [getUsedBytes <path>]");
+    System.out.println("       [getCapacityBytes <path>]");
```

Comment:
what is your suggestion for this?  The user should give a path like tachyon://<ip>:<port>/, do you want me to add this hint to help?

Thanks